apache-spark/spark3.0/image-classification/build.gradle

```diff
@@ -1,32 +1,46 @@
 plugins {
     id 'scala'
     id 'application'
+    id 'com.github.johnrengelman.shadow' version '7.0.0'
 }
 
 group "com.example"
 version "1.0-SNAPSHOT"
 
 repositories {
     mavenCentral()
+    mavenLocal()
 }
 
 dependencies {
-    implementation platform("ai.djl:bom:0.12.0")
-    implementation "org.apache.spark:spark-sql_2.12:3.0.1"
-    implementation "org.apache.spark:spark-mllib_2.12:3.0.1"
-    implementation "ai.djl:api"
-
-
-    runtimeOnly "ai.djl.pytorch:pytorch-model-zoo"
-    runtimeOnly "ai.djl.pytorch:pytorch-native-auto"
+    implementation "org.apache.spark:spark-core_2.12:${spark_version}"
```

Comment:
spark extension should have covered spack dependencies

Reply:
OK